DOF3_cubes:
  queries_number: 10
  query_0:
    goal_configuration:
    - 1.385647642233409
    - 0.6003216404601932
    - 0.18478799961190662
    - 0.8560989328404889
    - 0.13408689427133647
    - -0.6380470153793693
    start_configuration:
    - -0.03567181934416297
    - 0.5023762957289815
    - -1.1493491211013869
    - -0.9730483468852937
    - 0.903925664416328
    - 0.03247194920293994
  query_1:
    goal_configuration:
    - -0.009912887016870098
    - 1.0777508558057247
    - -0.9372069098392501
    - -1.421688316021487
    - 0.1246605793341995
    - 0.8452949284041302
    start_configuration:
    - 0.037143983582407225
    - 1.4687359043838455
    - -1.1808543264264242
    - 1.1582941582717001
    - -0.3212503667972981
    - -1.3598372106805443
  query_2:
    goal_configuration:
    - -1.2705435771184042
    - 1.4592680695401505
    - 1.3316390842359513
    - -1.3581152343096212
    - 1.2815320486413315
    - -0.4148046875456348
    start_configuration:
    - -0.5516458292456343
    - -0.26610463676154605
    - 0.6736375767488034
    - 0.32663720729704937
    - -1.3876004939496518
    - 0.6288764396013693
  query_3:
    goal_configuration:
    - -0.22453533392753444
    - -0.4934595126930623
    - 1.4027109815983103
    - 0.9565282385498286
    - -0.7057259274534881
    - 1.4422972811724992
    start_configuration:
    - -0.9088399454299361
    - 1.105197511659935
    - -0.8335810489688068
    - -0.6919409359222278
    - 0.7237127660052851
    - 0.9903097446089608
  query_4:
    goal_configuration:
    - -0.6381230492614209
    - -1.3022286983812228
    - 1.5244019385432825
    - -0.08820526677947482
    - -0.20883537010643627
    - 1.4778338607385753
    start_configuration:
    - -0.8151109584376216
    - 0.8637732208047064
    - -1.0574573491953312
    - 0.24294969592709093
    - 1.2915128934821112
    - -1.3068436071390286
  query_5:
    goal_configuration:
    - -1.081024402331002
    - 1.0382583843592552
    - -1.4537543981350958
    - 1.5397079880751667
    - -1.3440192995449527
    - -1.001534763955325
    start_configuration:
    - 0.4146685722090304
    - 1.1088471782287581
    - -1.2451030050365255
    - 0.5180868832860142
    - -0.014328240115754287
    - -1.4987206398027018
  query_6:
    goal_configuration:
    - -0.2680186405973508
    - 0.4669074560115114
    - -0.7432045726155863
    - 1.4640656138068067
    - 0.13751223499439647
    - 0.021742309455200948
    start_configuration:
    - -0.3457278676657005
    - 0.37034484346993257
    - -1.5676304876742884
    - -1.146092646870017
    - -0.8377514994995668
    - 0.43359159786589463
  query_7:
    goal_configuration:
    - 0.5637040584582838
    - 0.8386083333536984
    - 1.4674696068478748
    - -1.3333095101790502
    - 1.1610896096158771
    - 1.0513747476188466
    start_configuration:
    - 0.8932149000132454
    - 0.13252492713388064
    - 0.907161520232074
    - 0.4035677285654471
    - -0.1437060734644533
    - -0.8414261491103098
  query_8:
    goal_configuration:
    - -0.16951058164816346
    - -0.6653739164795727
    - -0.23483176531456418
    - -0.026220079562813137
    - 0.8918904184319081
    - 1.0743725333796814
    start_configuration:
    - 1.1932973200857637
    - -0.4728344334617256
    - -0.38670798179134724
    - 0.788642122239992
    - -1.0138917392933742
    - 0.4010615480041131
  query_9:
    goal_configuration:
    - -0.8634233923863619
    - 0.043765694407001066
    - -0.3848896471699699
    - 1.5503859252512455
    - 0.5792012306381016
    - -0.2452832393165678
    start_configuration:
    - -0.08866225127186622
    - -0.7755536283664405
    - 1.225565970013663
    - 0.08865140444040298
    - -0.23806922814957798
    - 0.2942169123893603
group_name: manipulator
pipeline_ids:
- dirrt
- ompl
planner_ids:
  dirrt:
  - mixed_strategy
  - mixed_strategy_extend
  - informed_rrt
  ompl:
  - BiTRRT
  - RRTstar
  - BFMT
  - LazyPRMstar
planning_times:
- 0.1
- 0.2
- 0.3
- 0.4
- 0.5
- 0.6
- 0.7
- 0.8
- 0.9
- 1.0
- 1.5
- 2.0
- 3.0
- 4.0
- 5.0
queries_number: 10
query_prefix: DOF3_cubes
repetitions: 30
